astrokat/simulate.py

```diff
@@ -147,74 +183,138 @@ def __exit__(self, type, value, traceback):
         now = timestamp2datetime(self.time)
         simobserver.date = ephem.Date(now)
-        user_logger.info('Slewed to %s', target.name)
+        user_logger.info("Slewed to %s", target.name)
         time.sleep(duration)
         now = timestamp2datetime(self.time)
         simobserver.date = ephem.Date(now)
-        user_logger.info('Tracked %s for %d seconds', target.name, duration)
+        user_logger.info("Tracked %s for %d seconds", target.name, duration)
         self.katpt_current = target
         return True
 
-    def raster_scan(self, target,
-                    num_scans=3,
-                    scan_duration=30.0,
-                    scan_extent=6.0,
-                    scan_spacing=0.5,
-                    scan_in_azimuth=True,
-                    projection='zenithal-equidistant',
-                    announce=True):
+    def raster_scan(
+        self,
+        target,
+        num_scans=3,
+        scan_duration=30.0,
+        scan_extent=6.0,
+        scan_spacing=0.5,
+        scan_in_azimuth=True,
+        projection="zenithal-equidistant",
+        announce=True,
+    ):
+        """Simulate raster scan.
+
+        Parameters
+        ----------
+        target: katpoint.Target object
+        num_scans: Number of scans
+        scan_extent: Time taken during scan
+        scan_spacing: time between scans
+        scan_in_azimuth: Scan in azimuth direction
+        projection: projection
+        announce:
+
+        """
         duration = scan_duration * num_scans
         time.sleep(duration)
         now = timestamp2datetime(self.time)
         simobserver.date = ephem.Date(now)
         return True
 
-    def scan(self, target,
-             duration=30.0,
-             start=(-3.0, 0.0),
-             end=(3.0, 0.0),
-             index=-1,
-             projection='zenithal-equidistant',
-             announce=True):
+    def scan(
+        self,
+        target,
+        duration=30.0,
+        start=(-3.0, 0.0),
+        end=(3.0, 0.0),
+        index=-1,
+        projection="zenithal-equidistant",
+        announce=True,
+    ):
+        """Simulate scan functionality during observations.
+
+        Parameters
+        ----------
+        target: katpoint.Target object
+        duration: Duration of scan
+        start:
+        end:
+        index:
+        projection:
+        announce:
+
+        """
         time.sleep(duration)
         now = timestamp2datetime(self.time)
         simobserver.date = ephem.Date(now)
         return True
 
     def slew_time(self, target):
-        slew_speed = 2.  # degrees / sec
+        """Get slew time.
+
+        How long in seconds, it took to the antennas to move
+        from previous target to the next
+
+        Parameters
+        ----------
+         target: str
+                 a comma-separated description which
+                 contains parameters such as the target name,
+                 position, flux model.
```

Comment:
No.  Implementation is assuming this is `katpoint.Target` object, not `str`.
E.g., usage of `target.body`.